tests/integration/test_final_bug_with_pk_columns_loading/test.py

```diff
@@ -0,0 +1,48 @@
+import pytest
```

Comment:
This could be a functional test.

Reply:
But the test restarts ClickHouse server.

Reply:
yep, I'm not aware of a way to trigger index optimisation w/o server restart